DEVELOPMENT_GUIDE.md

```diff
@@ -176,10 +176,10 @@ Because properties are fully customizable, it is important to use the final prop
 
 ## Custom Filters
 
-Ansible itself has a robust set of filters, but at times they do not fit cp-ansible’s needs. We have defined additional filters at `roles/confluent.variables/filter_plugins/filters.py`. In the below example we combine a custom filter `get_sasl_mechanisms` and our of the box ansible filters to set a variable:
+Ansible itself has a robust set of filters, but at times they do not fit cp-ansible’s needs. We have defined additional filters at `plugins/filter/filters.py`. In the below example we combine a custom filter `get_sasl_mechanisms` and our of the box ansible filters to set a variable:
```

Comment:
Reading thru this, I think we need to make a note in the dev guide that cp-ansible is now written as a collection. and that all filters must have confluent.platform in front of it. Itll be good practice to say that our standards are documented if anyone ever questions why something isnt working

Reply:
Create an issue for it? So we don't forget it.

Reply:
Added a comment into the issue to track docs: https://github.com/confluentinc/cp-ansible/issues/613#issuecomment-889131119